sql/upgrade/sl3.0-1.5.sql

```diff
@@ -1006,6 +1006,7 @@ SELECT setval('business_unit_id_seq', max(id)) FROM business_unit;
 --    SELECT MAX(CAST(???number AS NUMERIC))+1 FROM SL30.??? WHERE ???number ~ '^[0-9]+$'
 --) WHERE setting_key = 'paynumber';
 
+INSERT INTO defaults(setting_key,value) VALUES('default_country', :default_country);
```

Comment:
Isn't this unrelated to the change at hand?

Reply:
Nope, it is.